src/common/res/features/export-transactions/main.js

```diff
@@ -75,7 +75,23 @@
         downloadFile(args);
       }
 
+      // Sort an Array of Objects by key
+      function sortByKey(transactions, key, order) {
 
+        // Simple comparator
+        function compare(a, b) {
+          if (a[key] < b[key]) return -1;
+          if (a[key] > b[key]) return 1;
+          return 0;
+        }
+
+        transactions = transactions.copy();
+        transactions.sort(compare);
+        if (order === 'DESC') {
+          transactions.reverse
```

Comment:
You are right, although that's orthogonal to itemizing split transactions. Currently the exported CSV is in forward chronological order, but it seems the intention was for it to be backwards chronological order. Do you want me to fix that here?